flink-runtime/src/main/scala/org/apache/flink/runtime/minicluster/LocalFlinkMiniCluster.scala

```diff
@@ -350,23 +350,18 @@ class LocalFlinkMiniCluster(
 
   def setMemory(config: Configuration): Unit = {
     // set this only if no memory was pre-configured
-    if (config.getInteger(ConfigConstants.TASK_MANAGER_MEMORY_SIZE_KEY, -1) == -1) {
+    if (config.getLong(TaskManagerOptions.MANAGED_MEMORY_SIZE) == -1L) {
```

Comment:
I'm tempted to replace these with a `config.contains()` call but then the user would not be able to explicitly provide `-1` as the config option's value which is possible and valid at the moment. I don't really want to break existing behaviour though.
What do you think?